CLUE_BBQ/clue_bbq.py

```diff
@@ -0,0 +1,148 @@
+# Adafruit BBQ display works with ibbq protocol-based BLE temperature probes
+
+
+import time
+
+import displayio
+import _bleio
+import adafruit_ble
+from adafruit_ble.advertising.standard import ProvideServicesAdvertisement
+from adafruit_ble_ibbq import IBBQService
+from adafruit_clue import clue
+from adafruit_display_shapes.circle import Circle
+from adafruit_display_text import label
+from adafruit_bitmap_font import bitmap_font
+
+clue.display.brightness = 1.0
+homescreen_screen = displayio.Group(max_size=3)
+temperatures_screen = displayio.Group(max_size=2)
+
+# define custom colors
+GREEN =   0x00d929
+BLUE =    0x0000ff
+RED =     0xff0000
+ORANGE =  0xff6a00
+YELLOW =  0xffff00
+PURPLE =  0xe400ff
+BLACK =   0x000000
+WHITE =   0xffffff
+BURNT =   0xbb4e00
+
+unit_mode = False  # set the temperature unit_mode. True = centigrade, False = farenheit
+
+# background fill
+color_bitmap = displayio.Bitmap(120, 120, 1)
+color_palette = displayio.Palette(1)
+color_palette[0] = BURNT
+bg_sprite = displayio.TileGrid(color_bitmap, x=120, y=0, pixel_shader=color_palette)
+homescreen_screen.append(bg_sprite)
+
+clue_color = [GREEN, BLUE, RED, ORANGE, YELLOW, PURPLE]
+
+outer_circle = Circle(120, 120, 119, fill=BLACK, outline=BURNT)
+homescreen_screen.append(outer_circle)
+
+
+title_font = bitmap_font.load_font("/font/GothamBlack-50.bdf")
+title_font.load_glyphs("BQLUE".encode('utf-8'))
+title_label = label.Label(title_font, text="BBQLUE", color=clue.ORANGE, max_glyphs=15)
+title_label.x = 12
+title_label.y = 120
+homescreen_screen.append(title_label)
+
+clue.display.show(homescreen_screen)
+
+
+temp_font = bitmap_font.load_font("/font/GothamBlack-25.bdf")
+temp_font.load_glyphs("0123456789FC.-<".encode('utf-8'))
+
+my_labels_config = [(0, "", GREEN, 2, 100),
+                    (1, "", BLUE, 2, 150),
+                    (2, "", RED, 2, 200),
+                    (3, "", ORANGE, 135, 100),
+                    (4, "", YELLOW, 135, 150),
+                    (5, "", PURPLE, 135, 200)
+                   ]
+
+my_labels = {}  # dictionary of configured my_labels
+
+text_group = displayio.Group(max_size=8, scale=1)
+
+for label_config in my_labels_config:
+    (name, text, color, x, y) = label_config  # unpack a tuple into five var names
+    # old way sould have been: name = label_config[0], text = label_config[1]
```

Comment:
typo in comment ("sould"), and maybe this comment should just be removed?

Reply:
removing it